go/consensus/tendermint/abci/mux.go

```diff
@@ -268,6 +268,11 @@ func (a *ApplicationServer) State() api.ApplicationQueryState {
 	return a.mux.state
 }
 
+// GetLastRetainedVersion returns the earliest retained version.
+func (a *ApplicationServer) GetLastRetainedVersion(ctx context.Context) uint64 {
+	return a.mux.state.GetLastRetainedVersion()
```

Comment:
Could we avoid this separate method by moving `GetLastRetainedVersion` to `api.ApplicationQueryState` (or `api.ApplicationState` and make `State()` return the latter)?